common/membership/ringpop/factory_test.go

```diff
@@ -199,6 +203,10 @@ func (s *RingpopSuite) TestRingpopInvalidTLS() {
 	s.Error(runRingpopTLSTest(&s.Suite, s.insecureFactory, s.serverTLSFactoryB))
 }
 
+func (s *RingpopSuite) TestRingpopIPv6TLS() {
+	s.Error(runRingpopTLSTest(&s.Suite, s.serverIPv6TLSFactoryA, s.serverIPv6TLSFactoryB))
```

Comment:
Why do we expect this to cause an error? What about a test for if the TLS config is valid for IPV6?

Reply:
Its not so much that we expect it to error as much as that it confirms that tchannel supports ipv6. the question "are we sure ringpop supports ipv6" was one that was a little bit difficult to answer without just starting up a server using ipv6.

The test itself exercises `ringpop.factory.getListenIP` with `BindOnIP` set to an ipv6 address via `getTChannel()`, confirms that tchannel `Ping` works with ipv6, and also confirms that `tls.Dial` can make a tls connection over ipv6.

Basically it's an easy way to add one test and get some assurance about a lot of codepaths and their capability of handling an ipv6 address.

If there's a better way to do that I'm open.

Reply:
interestingly this is failing in the build pipeline - i guess because the runner doesn't support ipv6?

Reply:
so two questions -
* do we want to be able to run tests that require ipv6
* do we want to narrow the scope of the test (if so, to what?)

Reply:
oh man - yeah i read this wrong. the intention is `NoError` here